Add page for updated PCR upgrade info (#20729)
cockroachdb/docs: src/current/v25.4/upgrade-with-pcr.md

---
title: Upgrade with Physical Cluster Replication Enabled
summary: Upgrade your primary and standby clusters when using PCR.
toc: true
docs_area: manage
---

> When [**physical cluster replication (PCR)**]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) is enabled, you must use the process on this page to upgrade your clusters. This process ensures that the standby cluster safely upgrades before the primary cluster, preventing any version incompatibility. You cannot replicate data from a cluster on a newer version to a cluster on an older version. 

Reviewer: ```suggestion
To safely upgrade a cluster with [**physical cluster replication (PCR)**]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}), you must upgrade the standby cluster before the primary cluster to avoid version incompatibility. You cannot replicate data from a cluster on a newer version to a cluster on an older version. 

This page describes the process for safely upgrading a cluster that has PCR enabled.
```

    {{site.data.alerts.callout_info}}
    If you need to [roll back]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-major-version-upgrade) an upgrade, you must do so before the upgrade has been finalized.
    {{site.data.alerts.end}}

1. [Upgrade the binaries]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#perform-a-major-version-upgrade) on the primary cluster. Replace the binary on each node of the cluster and restart the node.

1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#finalize-a-major-version-upgrade-manually) the upgrade on the primary cluster's SystemVC. 

    {{site.data.alerts.callout_info}}
    If you need to [roll back]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#roll-back-a-major-version-upgrade) an upgrade, you must do so before the upgrade has been finalized. Rolling back the upgrade on the primary cluster does not roll back the standby cluster.
    {{site.data.alerts.end}}

1. [Finalize]({% link {{ page.version.version }}/upgrade-cockroach-version.md %}#finalize-a-major-version-upgrade-manually) the upgrade on the primary cluster's AppVC. 

>     Upgrading the primary cluster's AppVC also upgrades the standby cluster's AppVC, since it replicates from the primary.

Reviewer: I'm not finding any useful descriptions of how to do this, and the only search result for "AppVC" is this https://www.cockroachlabs.com/docs/stable/read-from-standby.html#how-the-read-from-standby-feature-works

Should definitely include a code snippet, and we may be missing other content on how to connect to the AppVC. Also as mentioned above, naming should be consistent and I see "application virtual cluster" more than "AppVC"
